### PR TITLE
Fixes to wiki not-found pages and chatroom button toggle

### LINKED ISSUE #1426: When accessing a page that doesn't exist, the page that shows has two footers?
### What happened just before the problem occurred

Playing around with cloned copy of PublicLab, realized the double footer bug I found was also on the actual PublicLab website. There's also two navbars, one that covers the other one? But it doesn't show, so I guess it's not really something that needs fixing. But the two footers appear one after another.

screen shot 2017-05-19 at 8 10 05 pm

How are _headers and _footers attached in plots2? I assumed it would be something like <%=render layouts/header> and <%= render layouts/footer>, but that doesn't seem to be the case. On my local copy I just deleted the 404.html header and footer to avoid replicating them.

### Relevant URLs
(any URL that doesn't resolve to a page)
https://publiclab.org/methodssdf

### PublicLab.org username
doesn't matter

### Browser, version, and operating system
doesn't matter

### LINKED ISSUE #1427: Multiple chatroom windows show, after chatroom icon is opened, closed, then opened again
### What happened just before the problem occurred
Chatroom icon was clicked once to open, clicked again to close, and clicked one more time. 
screen shot 2017-05-19 at 10 00 49 pm
^ You can see the second box here.

### Relevant URLs
https://publiclab.org/

### PublicLab.org username
n/a

### Browser, version, and operating system
n/a

### PR DESCRIPTION
**Pull request by ananyo2012**
Fixes #1426, fixes #1427

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by ananyo2012** [comment]
@jywarren Can you Review these fixes ?

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -154,7 +155,7 @@
       <% unless params[:action] == "register" || params[:action] == "signup" %>
         <ul class="nav navbar-nav navbar-right">
           <li class="dropdown">
-            <a onClick="$('#chat').toggle();$('#chat').append('<iframe width=\'400px\' height=\'500px\' style=\'border:0;\' src=\'https://chat.publiclab.org\'></iframe>');">
+            <a onClick="$('#chat').toggle();">
```
Comment (jywarren) [inline app/views/layouts/_header.html.erb]: Hi, @ananyo2012 !! great to see you. Here, we were trying not to actually instantiate the iframe until it's clicked. This change reverts that, so there could be some performance issues? (that was the original concern, anyways). Do you think we could set a local variable `true` on toggling, and only append if the variable is `!= true`?

Comment (ananyo2012) [inline app/views/layouts/_header.html.erb]: Ok I will change it. But seems a lot of inline javascript here. Will try to move this to any suitable javascript  file.

**Comment by ananyo2012** [comment]
@jywarren Made the suggested changes

**Comment by jywarren** [comment]
:balloon:

**Pull request merged**